Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -13,5 +21,40 @@ public IEnumerable<Book> GetBooks()
         {
             return databaseConnection.Query<Book>("SELECT * FROM Books");
         }
+
+        public Book GetCopies(string isbn)
```

Comment:
This should probably return a optional book, just in case a book with this isbn isn't found. You should handle that error case in the controller 🙂